src/core/sys/posix/sys/stat.d

```diff
@@ -534,6 +534,83 @@ version( CRuntime_Glibc )
         else
             static assert(stat_t.sizeof == 128);
     }
+    else version (SPARC64)
+    {
+        private
+        {
+            alias __dev_t = ulong;
+            alias __ino_t = c_ulong;
+            alias __ino64_t = ulong;
+            alias __mode_t = uint;
+            alias __nlink_t = uint;
+            alias __uid_t = uint;
+            alias __gid_t = uint;
+            alias __off_t = c_long;
+            alias __off64_t = long;
+            alias __blksize_t = c_long;
+            alias __blkcnt_t = c_long;
+            alias __blkcnt64_t = long;
+            alias __timespec = timespec;
+            alias __time_t = time_t;
+        }
+        struct stat_t
+        {
+            __dev_t st_dev;
+            ushort __pad1;
+            __ino_t st_ino;
```

Comment:
Looking at glibc 2.23 I see
```c
#if __WORDSIZE == 64 || !defined __USE_FILE_OFFSET64
    unsigned short int __pad1;
    __ino_t st_ino;			/* File serial number.	*/
#else
    __ino64_t st_ino;			/* File serial number.	*/
#endif
```
in `sysdeps/unix/sysv/linux/sparc/bits/stat.h`, so the above should be


```d
static if(!__USE_FILE_OFFSET64)
{
    ushort __pad1;
    __ino_t st_ino;
}
else
{
    __ino64_t st_ino;
}
```